ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-ctas-materialized.json

```diff
@@ -794,6 +794,330 @@
         ]}
       ]
     },
+    {
+      "name": "windowed - select star with HAVING filter - AVRO",
```

Comment:
are these related to this PR? I thought the problem was only with multi-column keys. I guess is it possible to window by a multi-column key?

Reply:
We do not have test cases for AVRO and protobuf formats. Add test cases for those formats and add a multi-column key test.